src/partials/header.html

```diff
@@ -0,0 +1,90 @@
+<header class="header-bg  ">
+  <div class="header-container">
+    <!-- <div class="header-logo"> -->
+      <a class="header-logo link" href="./index.html">
+        <svg class="header-logo-icon" width="127" height="16" aria-label="header minimal logo">
+          <use href=""></use>
+        </svg>
+              </a>
+      <!-- </div> -->
+      <nav class="header-nav">
+        <ul class="header-nav-menu list">
+          <li>
+            <a href="" class="header-nav-menu-item link">About us</a>
```

Comment:
class in link (<a>) needs small corrections, as menu-item generally should be added on the <li> element, so you can name the <a> element as header-nav-menu-link, for example